cat.recovery/10_basic is missing a recovery stage

It looks like cat.recovery/10_basic is missing one of the constants from `RecoveryState`.


```
FAILURE 0.29s J3 | Rest2IT.test {yaml=cat.recovery/10_basic/Test cat recovery output} <<<
   > Throwable #1: java.lang.AssertionError: field [$body] was expected to match the provided regex but didn't
   > Expected: ^
   > (
   >   index1      \s+
   >   \d          \s+                                 # shard
   >   \d+         \s+                                 # time
   >   (store|replica|snapshot|relocating)     \s+   # type
   >   (init|index|start|translog|finalize|done) \s+   # stage
   >   [-\w./]+    \s+                                 # source_host
   >   [-\w./]+    \s+                                 # target_host
   >   [-\w./]+    \s+                                 # repository
   >   [-\w./]+    \s+                                 # snapshot
   >   \d+         \s+                                 # files
   >   \d+\.\d+%   \s+                                 # files_percent
   >   \d+         \s+                                 # bytes
   >   \d+\.\d+%   \s+                                 # bytes_percent
   >   \d+         \s+                                 # total_files
   >   \d+         \s+                                 # total_bytes
   >   \d+         \s+                                 # translog
   >   -?\d+\.\d+% \s+                                 # translog_percent
   >   -?\d+       \s+                                 # total_translog
   >   \n
   > )+
   > $
   >      but: was "index1 0 3  store   done         local local n/a n/a 0 0.0%   0   0.0%   0 0   0 100.0% 0 \nindex1 0 48 replica verify_index local local n/a n/a 1 100.0% 130 100.0% 1 130 0 0.0%   1 \n"
   > 	at __randomizedtesting.SeedInfo.seed([ED06E3B0DDBF01F2:6552DC6A73436C0A]:0)
   > 	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
   > 	at org.elasticsearch.test.rest.section.MatchAssertion.doAssert(MatchAssertion.java:55)
   > 	at org.elasticsearch.test.rest.section.Assertion.execute(Assertion.java:69)
   > 	at org.elasticsearch.test.rest.ESRestTestCase.test(ESRestTestCase.java:373)
   > 	at java.lang.Thread.run(Thread.java:745)
  2> NOTE: leaving temporary files on disk at: /home/manybubbles/Workspaces/Elasticsearch/elasticsearch/core/target/J3/temp/org.elasticsearch.test.rest.Rest2IT_ED06E3B0DDBF01F2-001
  2> NOTE: test params are: codec=Asserting(Lucene53): {}, docValues:{}, sim=RandomSimilarityProvider(queryNorm=true,coord=crazy): {}, locale=it_IT, timezone=Australia/Currie
  2> NOTE: Linux 3.19.0-26-generic amd64/Oracle Corporation 1.8.0_45-internal (64-bit)/cpus=12,threads=1,free=388863912,total=517472256
  2> NOTE: All tests run in this JVM: [UpdateMappingOnClusterIT, Rest2IT]
```

Reproduction command:
```
mvn verify -Pdev -Dskip.unit.tests -pl org.elasticsearch:elasticsearch -Dtests.seed=ED06E3B0DDBF01F2 -Dtests.class=org.elasticsearch.test.rest.Rest2IT -Dtests.method="test {yaml=cat.recovery/10_basic/Test cat recovery output}" -Des.logger.level=ERROR -Dtests.assertion.disabled=false -Dtests.security.manager=true -Dtests.heap.size=512m -Dtests.locale=it_IT -Dtests.timezone=Australia/Currie
```

I'm not sure which versions this effects.

--- a/rest-api-spec/src/main/resources/rest-api-spec/test/cat.recovery/10_basic.yaml
+++ b/rest-api-spec/src/main/resources/rest-api-spec/test/cat.recovery/10_basic.yaml
@@ -28,8 +28,8 @@
                 index1      \s+
                 \d          \s+                                 # shard
                 \d+         \s+                                 # time
-                (store|replica|snapshot|relocating)     \s+   # type
-                (init|index|start|translog|finalize|done) \s+   # stage
+                (store|replica|snapshot|relocating)     \s+     # type
+                (init|index|verify_index|translog|finalize|done) \s+ # stage
                 [-\w./]+    \s+                                 # source_host
                 [-\w./]+    \s+                                 # target_host
                 [-\w./]+    \s+                                 # repository

